### DISCUSSION TITLE
Install Coqui TTS on web server

### DISCUSSION THREAD

**Discussion by masai2k** [coqui-ai/TTS#2814] [General Q&A]
Hi, I want to install coqui TTS online, on a web server. I have access to pythoneverywhere, but I really don't know how to install coqui to the server. Could anyone help me, or link me to a tutorial to do that?

Thanks a lot to this fantastic community!

Massimo

**Comment by MentalGear** [comment]
Maybe try running it via docker on a webserver ?
